contracts/metagov/BalancerGaugeStaker.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "../pcv/PCVDeposit.sol";
+import "./utils/LiquidityGaugeManager.sol";
+import "../external/balancer/IBalancerMinter.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/// @title Deposit that can stake in Balancer gauges
+/// @author Fei Protocol
+contract BalancerGaugeStaker is PCVDeposit, LiquidityGaugeManager {
+    using SafeERC20 for IERC20;
+
+    address public constant BALANCER_GAUGE_CONTROLLER = 0xC128468b7Ce63eA702C1f104D55A2566b13D3ABD;
+    address public constant BALANCER_MINTER = 0x239e55F427D44C3cc793f49bFB507ebe76638a2b;
```

Comment:
this should not be immutable because balancer could change it

Reply:
recommend having both settable

Reply:
ok there is a setter for the gauge controller, will add a setter for the minter

Reply:
I'm removing the constants & putting these as constructor args